src/css/styles.css

```diff
@@ -460,7 +460,7 @@ a {
   list-style: none;
   display: flex;
   flex-wrap: wrap;
-  max-width: 1320px;
+  max-width: 1760px;
```

Comment:
I'd rather add breakpoints here and make it two rows on desktop than having it that wide.

Reply:
Update this. Can you have another look?

Reply:
Yup, good like that! Thanks :)